### ISSUE TITLE
Markdown tables are not supported?

### ISSUE THREAD

**Issue by afarran** [lunarmodules/LDoc#196]
Having ldoc comments in this markdown format:
Item 1,1 | Item 1,2
---------- | ------------
Item 2,1 | Item 2,2
Item 3,1 | Item 3,2

does not produce the following expected HTML table output:

| Item 1,1 | Item 1,2 |
| --- | --- |
| Item 2,1 | Item 2,2 |
| Item 3,1 | Item 3,2 |

Is this a bug? Is there a workaround other than embedding HTML table tags in the ldoc comments?

Tested with Ldoc v1.4.3.
**config.ld** has the following settings:
- no_summary=false
- not_luadoc=true
- format='markdown'
- plain=false
- boilerplate=false

**Comment by stevedonovan** [comment]
You certainly get a generated table with Discount (built-in markdown.lua doesn't support tables) but we don't have a nice style sheet that makes that table look pretty. Should be an easy fix.

**Comment by keneanung** [comment]
I've been also using ldoc 1.4.3 with the `config.ld` set to `format="discount"` and the following table:

```
...
Dependencies:

| Dependency          | Version |
|---------------------|---------|
| `IOE.Core`          | 2       |

### Purpose
...
```

But the output I get is http://ioe.gitlab.io/topics/IOE.TargettingREADME.md.html

I have no idea what I'm doing wrong, can you help?

**Comment by cloewen8** [comment]
I also had trouble getting tables to work. I used lua-discount, but it turns out that the current version of lua-discount does not support tables. I managed to fix it by using https://github.com/squeek502/lua-discount (a real pain to install with a lot of undocumented bugs). It seems to be by the same creator of the discount (not lua-discount) rock, but I can't verify that.

**Comment by alerque** [comment]
As noted, this is entirely up to the Markdown parser you use. There are several more available now, with Pandoc in the works (see #327) which will be another option besides discount that supports tables.
